### ISSUE TITLE
Add option to start up minified 

### ISSUE THREAD

**Issue by patricknelson** [dale-roberts/MouseUnSnag#7]
Along with #3, I'd love an option to have this start up mified to the system tray similar to what you get with [RBTray](https://github.com/benbuck/rbtray). For now, I'll be using this in conjunction with that tool and just manually right-clicking the "minimize" button every time I log in or reboot my computer 👍 

Thanks so much for this tool!

**Comment by patricknelson** [comment]
p.s. Alternatively a VBS file ([like the example here](https://superuser.com/questions/198525/how-can-i-execute-a-windows-command-line-in-background#answer-198530)) could be included, but this doesn't provide any easy UI (e.g. system tray icon) to terminate the application. 

**MouseUnSnag (hidden).vbs:**
```
Dim WinScriptHost
Set WinScriptHost = CreateObject("WScript.Shell")
WinScriptHost.Run Chr(34) & "MouseUnsnag" & Chr(34), 0
Set WinScriptHost = Nothing
```

You just have to be careful since `MouseUnSnag.exe` doesn't run as a single instance.

**Comment by EcurbNosneb** [comment]
Drag MouseUnSnag shortcut into the Startup folder.  Edit properties, Shortcut.  Select Run Minimized. Optionally change Icon to something nicer than the default.  I picked the old Windows butterfly as this great little program freed my mouse to move between my 4K monitor flanked by two 1K monitors. Where can I send money? <3

**Comment by patricknelson** [comment]
Just following up here to mention that this is no longer relevant, since the project has since moved to https://github.com/MouseUnSnag/MouseUnSnag and is the system tray capability is already available in releases circa 2.1 and later.
